pkgs/tools/backup/mydumper/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub
+, cmake, glib, mariadb, libressl
+, pcre, pkgconfig, zlib
+}:
+
+stdenv.mkDerivation rec {
+  version = "0.9.3";
+  name = "mydumper-${version}";
+
+  src = fetchFromGitHub {
+    owner  = "maxbube";
+    repo   = "mydumper";
+    rev    = "eef4d8291803e9ba57b4e58458cec3f4f7f0b203";
+    sha256 = "139v6707sxyslg7n1fii8b1ybdq50hbqhc8zf6p1cr3h2hhl6ns9";
+  };
+
+  buildInputs = [ cmake glib mariadb.client.dev libressl pcre pkgconfig zlib ];
+
+  meta = with stdenv.lib; {
```

Comment:
Are you willing to maintain this package?
If so please add yourself as a maintainer in https://github.com/NixOS/nixpkgs/blob/master/lib/maintainers.nix

Personally I'm very reluctant to merge new packages without a maintainer.